docs/data/toolpad/core/components/sign-in-page/PasskeySignInPage.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { AuthProvider, AppProvider, SignInPage } from '@toolpad/core';
+import { useTheme } from '@mui/material/styles';
+// preview-start
+const providers = [{ id: 'passkey', name: 'Passkey' }];
+// preview-end
+
+const signIn: (provider: AuthProvider) => void = async (provider) => {
```

Comment:
Would it be possible for the demo to still show some kind of feedback when we submit the form like the other ones do?